docs/terminal/eastsale.md

```diff
@@ -1,10 +1,247 @@
----
-  title: East Sale TCU
----
+# East Sale TCU
 
---8<-- "includes/abbreviations.md"
+## Positions
 
-Reserved.
+| Name               | ID      | Callsign             | Frequency        | Login Identifier    |
+| ------------------ | ------- | ------------------- | ---------------- | ------------------- |
+| **East Sale TMA**  | **ESL** | **East Sale Approach** | **123.300**      | **ES_APP**          |
 
-!!! note
-    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
\ No newline at end of file
+## Airspace
+
+ES TCU controls the airspace within R360, from `SFC` to `F210`. The TCU controller has control from the surface up; if circuit operations need to be handled by East Sale Tower, a release is required.
+
+Where traffic levels, particularly during military operations, necessitate, ES TCU can request the release of additional airspace sectors from the overlying enroute controller (e.g., Melbourne Center).
+
+### Activating Additional Airspace
+
+If operationally necessary, ES TCU may request the activation of `M301` airspace segments for military exercises to `F450`. Coordination with the overlying enroute controller is required to secure these airspace releases.
+
+## Logon Process
+
+Given the specific nature of the East Sale TCU airspace, surrounding controllers may not be fully aware of its boundaries. Upon logging on as East Sale TCU, it is essential to announce the activation status of R360/M301.
+
+> **Example:**
+> **ES TCU** -> **Melbourne Center**: "Request activation of R360 from SFC to F210 for military operations."
+
+## Circuit Procedures
```

Comment:
ES TCU, although they own the airspace by default, generally won't be handling circuits. This section can be removed and kept in the tower page. If the ES TCU controller finds themselves in the highly unusual situation where the ADC is online, but TCU is managing the circuit, they can find the info on the ADC SOPs

Reply:
I'll leave just the west sale ones in. But will remove all the duplicate TCU stuff